scalanet/src/io/iohk/scalanet/peergroup/UDPPeerGroup.scala

```diff
@@ -255,33 +256,33 @@ class UDPPeerGroup[M](val config: Config)(
         recipient: InetSocketAddress,
         nettyChannel: NioDatagramChannel
     ): Task[Unit] = {
-      log.debug("Sending message {} to peer {}", message, recipient)
-      Task.fromTry(codec.encode(message).toTry).flatMap { encodedMessage =>
-        val asBuffer = encodedMessage.toByteBuffer
-        toTask(nettyChannel.writeAndFlush(new DatagramPacket(Unpooled.wrappedBuffer(asBuffer), recipient, sender)))
-          .onErrorRecoverWith {
-            case _: IOException =>
-              Task.raiseError(new MessageMTUException[InetMultiAddress](to, asBuffer.capacity()))
-          }
-      }
+      Task.delay(logger.debug("Sending message {} to peer {}", message, recipient)) *>
```

Comment:
maybe as now we have chain of 3 flatmaps, maybe it is worth to use for comprehension here ?

Reply:
Sure, no probs.

Reply:
Done.